### PR TITLE
update ecosystem

### PR DESCRIPTION
**Pull request by skzhang1**
## Why are these changes needed?

Update the ecosystem page for Ollama, hf, and other potential integrations.

- [x] ollama
- [ ] hf   

👋  I'd love to hear other integrations that deserve to be added. If you have any other ideas, feel free to share them here!

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/docs/Ecosystem.md [hunk 1]
```diff
@@ -19,3 +19,11 @@ MemGPT enables LLMs to manage their own memory and overcome limited context wind
 [Microsoft Fabric](https://learn.microsoft.com/en-us/fabric/get-started/microsoft-fabric-overview) is an all-in-one analytics solution for enterprises that covers everything from data movement to data science, Real-Time Analytics, and business intelligence. It offers a comprehensive suite of services, including data lake, data engineering, and data integration, all in one place. In this notenook, we give a simple example for using AutoGen in Microsoft Fabric.
 
 - [Microsoft Fabric + AutoGen Code Examples](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_microsoft_fabric.ipynb)
+
+## Ollama + AutoGen
+
+![Ollama Example](img/ecosystem-ollama.png)
```
Comment (jackgerrits) [inline website/docs/Ecosystem.md:25]: I love the graphic but I really want to be able to copy and paste this. Can you use just the image and then use a code block?

**Review by jackgerrits — CHANGES_REQUESTED** [review]

File: website/docs/Ecosystem.md (hunk 1 above)
Comment (skzhang1) [reply website/docs/Ecosystem.md:25]: I changed it to its official logo, making the style consistent with other integrations.

**Comment by skzhang1** [comment]
Hi @victordibia, which page of the hf (a specific documentation or codebase) we should use for updating the ecosystem page?🤔

**Comment by sonichi** [comment]
Let's skip hf in this PR.

**Review by jackgerrits — APPROVED** [review]

**Pull request merged**